Bump version

hkollmann opened the pull request:
I found a lot of files where the version string was 5.1.0. So i run bumpqxversion.py to change it in all files to 6.0.0-alpha.
I also add some licence fields in package.json.
And third i moved the grunt stuff in /package.json to devDependencies

hkollmann committed:
bump version

I found a lot of files where the version string was 5.1.0. So i run bumpqxversion.py to change it in all files to 6.0.0-alpha

hkollmann committed:
add some more licence fields,
move grunt to devDependencies in /package.json

hkollmann committed:
forget one file

cajus commented on documentation/manual/source/conf.py:
```diff
@@ -56,17 +56,17 @@
 release = '6.0.0-alpha'
 # The current git branch (used for github links)
-git_branch = "master"
+git_branch = "bump_version"
```
Is that correct?

hkollmann replied:
changed it back

cajus commented on tool/admin/release/index.html:
```diff
@@ -209,42 +209,42 @@ <h2>Website Tests</h2>
           <h2>Website Lib</h2>
```
I'd vote for dropping the admin stuff and most parts of the tech documentation before 6.0. It does not get deployed and the technical information in there is outdated anyways.

johnspackman replied:
:+1

hkollmann replied:
+1 but we should have some technical information somewhere. It was painfull to figure this out. And it‘s not very obvious how you can make a lokal build to test the changes in the build toolchain

hkollmann replied:
But this is another sory.

hkollmann committed:
Revert git branch

hkollmann committed:
Do not bump git branch in config.py

hkollmann committed:
With this patch we can integrate qooxdoo-sdk into qooxdoo-cli as an node plugin.
If you then run node install you will get qooxdoo-sdk version 5.0.2.

As developer you can then do a
git link ../qooxdoo
for linking the current working copy.

The patch is needed because npm link works with the module name.

I checked the current build process and found no issues which depends on the old module name.
After accept i will prepare qooxdoo-cli to use the qooxdoo-sdk

hkollmann committed:
Revert "With this patch we can integrate qooxdoo-sdk into qooxdoo-cli as an node plugin."

This reverts commit 29f19dd62a920169f1602cde3c49f19205ac8495.

hkollmann commented:
It´s documentation. So 2 approvals are enough.

level420 commented:
@hkollmann yes it is a doc only PR, but from our dev rules you’re not allowed to merge your own PR. For my PRs I’m allways asking in the core team for someone who is willing to merge.

hkollmann commented:
Sorry for this next i will do so!

level420 commented:
@hkollmann you're right! it was never in the docs. Maybe we only talked about the restriction and we had it in practice without writing it down. Sorry about this.
